requirements.txt

```diff
@@ -5,4 +5,10 @@ djangorestframework==3.9.0
 PyJWT==1.7.1
 six==1.12.0 
 psycopg2-binary==2.7.6.1
-pycodestyle==2.4.0
\ No newline at end of file
+pycodestyle==2.4.0
+django-heroku==0.3.1
+dj-database-url==0.5.0
+pytz==2018.9
+whitenoise==4.1.2
+gunicorn==19.9.0
+django-heroku==0.3.1
```

Comment:
We have django-heroku twice in this file

Reply:
> Are we supposed to have DEBUG=True in production?

We'll change this when we're ready to ship for production?